gutenberg-ramp.php

```diff
@@ -148,6 +162,11 @@ function gutenberg_ramp_require_gutenberg() {
  */
 gutenberg_ramp_require_gutenberg();
 
+if ( Gutenberg_Ramp_Compatibility_Check::should_check_compatibility() ) {
+	$ramp_compatibility = new Gutenberg_Ramp_Compatibility_Check();
```

Comment:
Should we skip loading the rest of Ramp if the versions don't match?

Reply:
I don't think it's necessary. If Ramp isn't compatible - it doesn't do anything apart from attaching itself to 2 filters that will never be called, but it still does allow you to interact with it the you would normally.

I'm hesitant to change this because this is an edge case we're talking about. Refactoring the plugin to activate on a condition (`if can_activate()`) just because of an edge seems like a bit of overkill.

Ramp checks for whether it should be checking for compatibility at all, and if it should - it checks the compatibility and adds a notice - leaving the rest of the Ramp behavior as it was previously. It's simple, easy, and it works.